src/views/Home/components/Banners/FarmAuctionsBanner.tsx

```diff
@@ -44,7 +50,7 @@ const LeftWrapper = styled(Flex)`
   flex-direction: column;
   justify-content: center;
 
-  ${({ theme }) => theme.mediaQueries.sm} {
+  ${({ theme }) => theme.mediaQueries.md} {
```

Comment:
If not too difficult - can you please make some fix for iPhone 5-kind screens (320px)? 🙂 
![Screenshot 2021-08-16 at 13 55 25](https://user-images.githubusercontent.com/81824236/129553297-e168d8bc-d015-4d4d-a96d-d9d9ec20afc1.png)

Reply:
Done